pkgs/applications/science/electronics/linux-gpib/linux-gpib-user/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchurl
+, autoconf
+, libtool
+, bison
+, flex
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib";
```

```suggestion
  pname = "linux-gpib-user";
```
otherwise pname is the same which is not that nice